animal-sniffer-signature/build.gradle.kts

```diff
@@ -0,0 +1,70 @@
+import ru.vyarus.gradle.plugin.animalsniffer.info.SignatureInfoTask
+import ru.vyarus.gradle.plugin.animalsniffer.signature.BuildSignatureTask
+
+plugins {
+    id("com.android.library")
+    id("ru.vyarus.animalsniffer")
+}
+
+android {
+    namespace = "otel.animalsniffer.signature"
+    compileSdk = (property("android.compileSdk") as String).toInt()
+}
+
+description = "Build tool to generate the Animal Sniffer Android signature"
+
+val signatureJar =
+    configurations.create("signatureJar") {
+        isCanBeConsumed = false
+        isCanBeResolved = false
+    }
+val signatureJarClasspath =
+    configurations.create("signatureJarClasspath") {
+        isCanBeConsumed = false
+        isCanBeResolved = true
+        extendsFrom(signatureJar)
+    }
+val generatedSignature =
+    configurations.create("generatedSignature") {
+        isCanBeConsumed = true
+        isCanBeResolved = false
+    }
+configurations.add(signatureJar)
+configurations.add(signatureJarClasspath)
+configurations.add(generatedSignature)
+
+dependencies {
+    signature("com.toasttab.android:gummy-bears-api-${project.property("android.minSdk")}:0.9.0@signature")
```

Comment:
What's all this about?

Reply:
this is the plugin that will be compiled against (pinned to the min version SDK) so the sniffer knows if there's code that's not compatible with